extension/src/popup/views/ViewPublicKey/index.tsx

```diff
@@ -63,37 +64,44 @@ export const ViewPublicKey = () => {
     <>
       <PopupWrapper>
         <div className="ViewPublicKey">
-          <Formik initialValues={initialValues} onSubmit={handleSubmit}>
-            <div className="ViewPublicKey__header">
-              <BackButton />
-              {isEditingName ? (
-                <>
-                  <div className="ViewPublicKey--account-name-div"></div>
-
-                  <Form className="ViewPublicKey__form">
-                    <Field name="accountName">
-                      {({ field }: FieldProps) => (
-                        <Input
-                          autoComplete="off"
-                          id="accountName"
-                          placeholder={accountName}
-                          {...field}
-                        />
-                      )}
-                    </Field>
-                  </Form>
-                </>
-              ) : (
-                <>
-                  <div className="ViewPublicKey__account-name-display">
-                    {accountName}
-                  </div>
-                </>
-              )}
-              <div className="ViewPublicKey--account-name-div">
-                <EditNameButton />
+          <Formik
```

Comment:
max characters for account name
https://stellarorg.atlassian.net/browse/WAL-96?filter=10043

Reply:
one of the subtasks for this is adding a disable horizontal scroll: https://stellarorg.atlassian.net/browse/WAL-97?filter=10043

my thoughts are that _ideally_ there shouldn't be a place that causes the screen to grow horizontally outside the popup. But if we miss a spot, then we'd want to allow scroll so important content doesn't get cut off.
(eg. if we missed this account name max, and didn't allow scroll then the edit name button would be hidden and they'd be left unable to edit their name.)

wdyt?

Reply:
Yeah, agreed. In the case of a bug, I'd rather have the UI look a little ugly than cut off content. I'll comment on the ticket